bin/martinize2

```diff
@@ -977,8 +1023,21 @@ def entry():
             "The output topology will require files generated by " '"create_goVirt.py".'
         )
         vermouth.MergeAllMolecules().run_system(system)
+        # not ideal but we need it before making the Go sides
+        res_graph = vermouth.graph_utils.make_residue_graph(system.molecules[0])
```

Comment:
Will remove the message. 

It might work but the virtual sides are disconnected and at the end of the molecule. So if we make a residue graph after they are added it will be quite slow and they will get separated from the residue they are associated with. Using the old_resid argument we can probably get rid of it. BUT what we really SHOULD do is associate the virtual-sides with the residue. However, in the past this has lead to some problems with downstream usage of the Go model because people expect them to be in a certain format.

Reply:
I'm still not sure I understand. What's preventing you from building the residue graph the moment `GoButDecent.run_molecule` is called?
Also, I'm of the opinion vsites should have edges to their constructing atoms, but I think this was pulled out at some point --- can't remember why though.
And we can break all downstream considerations now, can't we?

Reply:
we do now